pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -460,7 +464,7 @@ pub mod pallet {
 			domain_address: DomainAddress,
 			message_hash: MessageHash,
 			router_id: T::RouterId,
-		) -> DispatchResult {
+		) -> DispatchResultWithPostInfo {
```

Comment:
This is now a `DispatchResultWithPostInfo` given that we can get the number of inbound sub-messages when processing.

Also, are we sure that we won't be adding support for batch messages coming from EVM? If so, we can remove this whole counter logic completely.

Reply:
I'm more on the way of: If we do not really need this now, let's keep things simple

Reply:
I'm also in favor of removing it and adding back later. Less maintenance in between.

Reply:
Cool, was asking because I wasn't sure if Solidity is already working on this or not. Will remove, thank you!